quarkchain/evm/specials.py

```diff
@@ -237,16 +237,32 @@ def proc_current_mnt_id(ext, msg):
 def proc_transfer_mnt(ext, msg):
     from quarkchain.evm.messages import apply_msg
 
-    # Data must be >= 96 bytes
-    if msg.data.size < 96:
-        return 0, 0, []
-    gascost = 3
-    if msg.gas < gascost:
+    # Data must be >= 96 bytes and static call not allowed
+    if msg.data.size < 96 or msg.static:
         return 0, 0, []
     to = utils.int_to_addr(msg.data.extract32(0))
     mnt = msg.data.extract32(32)
     value = msg.data.extract32(64)
     data = msg.data.extract_all(96)
+
+    num_zero_bytes = data.count(ascii_chr(0))
+    num_non_zero_bytes = len(data) - num_zero_bytes
+    # Intrinsic gas
+    gascost = (
```

Comment:
a second thought: as the caller already pays the intrinsic gas for the tx itself (including the origin data), should we add an another layer of intrinsic cost? @qizhou

Reply:
I think we don't need to pay the intrinsic gas here - intrinsic gas should be only used when the tx is initiated, but I haven't seen the intrinsic gas about a sub-message call.  Could you check it again?